compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -621,35 +624,47 @@ def start(self):
     def _event_loop(self):
         parent_identities: set[str] = set()
         if not is_privileged():
-            client_options = {
-                "local_compute_services": self._config.local_compute_services,
-                "environment": self._config.environment,
-                "app": get_globus_app_with_scopes(),
-            }
-            log.debug("Ascertaining user identity set (%s)", client_options)
 
-            gcc = GC.Client(**client_options)
-            ac = ComputeAuthClient(app=gcc.app)
-            try:
-                userinfo = ac.userinfo()
-                ids = userinfo["identity_set"]
-                parent_identities.update(ident["sub"] for ident in ids)
-                log.debug(
-                    "User-endpoint start requests are valid from identities: %s",
-                    parent_identities,
-                )
-                del gcc, client_options, ids
-                if not parent_identities:
-                    # Not a privileged user -- we require at least one identity
-                    # against which to match start endpoint requests.
-                    raise LookupError("No authorized identities found")
-
-            except Exception as exc:
-                msg = "Failed to determine identity set; try `whoami` command?"
-                log.error(f"({type(exc).__name__}) {exc}\n    {msg}")
-                log.debug("Stopping; failed to determine identities", exc_info=exc)
-                self._time_to_stop = True
-                return
+            def _get_identities():
```

Comment:
I like this organization, but consider naming this function `_get_globus_identities()` to clearly distinguish from local users.